hack/verify-staging-imports.sh

```diff
@@ -79,7 +79,7 @@ if grep -rq '// import "k8s.io/kubernetes/' 'staging/'; then
 	exit 1
 fi
 
-for EXAMPLE in vendor/k8s.io/client-go/examples/{in-cluster,out-of-cluster,third-party-resources}; do
+for EXAMPLE in vendor/k8s.io/client-go/examples/{authenticate-in-cluster,authenticate-out-of-cluster,third-party-resources}; do
```

Comment:
There are ways to configure the client without supplying the authentication information, so authentication doesn't always equal to configuration. So i think "configuration" is slightly better.

> It also helps with discovery as the letter A is listed first on GitHub.

We have your ToC now so we don't depend on the alphabetical order anymore :)